content/blog/2023-11-15.md

```diff
@@ -0,0 +1,8 @@
++++
+title = "2023-11-15"
+date = 2023-11-15
++++
+
+---
+
+Today is a Brazilian holiday. [Proclamation of the Republic day](https://en.wikipedia.org/wiki/Proclamation_of_the_Republic_(Brazil)).
```

Comment:
Do you mind re-adding _Today is Tuesday, day 318 of 365 (87.12%). We're in week 46 of 52 (88.46%)._  count to this and the other holidays ? Also to the sick day.